manifests/moco-config.pp

```diff
@@ -168,16 +151,12 @@
     $signing_allowed_ips             = [
         '10.134.68.32/32', # dev-master2
         '10.26.48.41/32',  # partner-repack1
-        '10.26.44.0/22',   # wintry.releng.scl3 (vlan 244)
-        '10.26.52.0/22',   # build.releng.scl3 (vlan 252)
-        '10.26.64.0/22',   # try.releng.scl3 (vlan 264)
```

Comment:
We can reduce this and `signing_new_token_allowed_ips` even further but I'll make a separate pull for that.

Reply:
👍